cmake/GosuConfig.cmake.in

```diff
@@ -106,8 +108,34 @@ endforeach()
 
 #message(FATAL_ERROR "blub")
 
+if(APPLE)
+    set(Gosu_APPLE_FRAMEWORKS "-framework OpenAL -framework IOKit -framework OpenGL -framework Carbon -framework Cocoa -framework AudioToolbox -framework ApplicationServices")
```

Comment:
How does this work? I haven't seen _APPLE_FRAMEWORKS variables anywhere else. Generally my issue right now is that I have no idea how to handle frameworks =/